C/c4CAPI.cc

```diff
@@ -278,19 +284,21 @@ bool c4coll_isValid(C4Collection* coll) noexcept {
 }
 
 C4CollectionSpec c4coll_getSpec(C4Collection *coll) noexcept {
+    // Unlike the others, this continues to return valid data even
+    // after invalidation, so skip the validity check
     return coll->getSpec();
```

Comment:
To be consistent, would you consider to check null?

Reply:
Yes certainly I will.  I also have another style that I think might be useful for the future in which we put some contract comments on the collection class, and reference them here when we don't check collection validity.  Then it can be easily apparent to the reader why the check is "missing"

Reply:
Done!